bedrock/settings/static_media.py

```diff
@@ -687,6 +687,13 @@
         ),
         'output_filename': 'css/persona-bundle.css',
     },
+    'personal': {
```

Comment:
This is a bit, but should the CSS bundle be called `firefox-personal` to match the JS bundle name?